tests/features/collection/propose.feature

```diff
@@ -62,6 +62,7 @@ Feature: Proposing a collection
     And I should see the heading "Ancient and Classical Mythology"
     And I should see a logo on the header
     And I should see a banner on the header
+    And I should not see "Thank you for proposing a collection. Your request is currently pending approval by the site administrator."
```

Comment:
This is a nice spot to assert that the message is not shown as we save the entity as "Save as draft".
However, the file is called propose.feature and the scenario mentions that we are proposing a collection.

This is not a problem with your implementation, it seems however that the test is not working as expected.
Can you fix the scenario in total? SImply instead of saving as draft, try to propose the collection instead.

Reply:
Sorry, but now that you fixed the form submit, I noticed that this is actually not working. Tested it through the UI.
The message is not shown when proposing a collection but should according to the requirements. So this step needs to be changed to a "And I should see...." and be fixed.

Reply:
Form is fixed and tests also fixed.